src/ps1/ps1_cardman.c

```diff
@@ -30,6 +32,7 @@ void ps1_cardman_init(void) {
     card_chan = settings_get_ps1_channel();
     if (card_chan < CHAN_MIN || card_chan > CHAN_MAX)
         card_chan = CHAN_MIN;
+    memset(card_game_id, 0, 0x10);
```

Comment:
this isn't needed, but if you want to keep it anyway use sizeof(card_game_id) instead of 0x10

Reply:
Changed it to sizeof. Init should always initialize all variables again - so its kept there